app/actions/views/thread.js

```diff
@@ -12,3 +12,30 @@ export function handleCommentDraftChanged(rootId, draft) {
         }, getState);
     };
 }
+
+export function handleCommentDraftSelectionChanged(rootId, cursorPosition) {
+    return {
+        type: ViewTypes.COMMENT_DRAFT_SELECTION_CHANGED,
+        rootId,
+        cursorPosition
+    };
+}
+
+export function insertToCommentDraft(rootId, value) {
+    return (dispatch, getState) => {
+        const {draft, cursorPosition} = getState().views.thread.drafts[rootId];
+
+        let nextDraft = `${value}`;
+        if (cursorPosition > 0) {
+            const beginning = draft.slice(0, cursorPosition);
+            const end = draft.slice(cursorPosition);
+            nextDraft = `${beginning}${value}${end}`;
+        }
+
+        dispatch({
```

Comment:
dispatch if it actually changed

Reply:
This function isn't being used anymore. I'll remove it.